editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/bean/AttributeTextFieldPropertyUICreator.java

```diff
@@ -31,12 +31,14 @@
  */
 public class AttributeTextFieldPropertyUICreator extends AbstractTextFieldParameterPropertyUICreator {
 
+	@SuppressWarnings("squid:S00107")
 	public AttributeTextFieldPropertyUICreator(DataBindingContext dbc, IObservableMap<String, String> modelMap, Eip eip, AbstractCamelModelElement camelModelElement, Parameter parameter,
 			Composite parent, TabbedPropertySheetWidgetFactory widgetFactory) {
 		super(dbc, modelMap, eip, camelModelElement, parameter, parent, widgetFactory, 
 				new AttributeTextParameterPropertyModifyListenerForAdvanced(camelModelElement, parameter));
 	}
 
+	@SuppressWarnings("squid:S00107")
```

Comment:
for this one too (as it is the same kind ^^), i think this is a "real" design issue.
I'm okay not to handle it right now but I would prefer to not ignore it by "suppressing the warning"